yt_dlp/extractor/redbee.py

```diff
@@ -69,6 +69,9 @@ def _get_formats_and_subtitles(self, asset_id, **kwargs):
                 fmts, subs = self._extract_m3u8_formats_and_subtitles(
                     format['mediaLocator'], asset_id, fatal=False)
 
+            for index in range(len(fmts)):
+                fmts[index]['has_drm'] = format.get('drm') is not None
```

Comment:
_extract_m3u8_formats_and_subtitles etc has it's own drm detection. Don't override it

```suggestion
            if format.get('drm'):
                for f in fmts:
                    f['has_drm'] = True
```